_posts/languages/php/2000-01-01-wordpress.md

```diff
@@ -1,78 +1,148 @@
+   ```bash
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Action": [
+            "s3:PutObject",
+            "s3:PutObjectAcl",
+            "s3:PutObjectAclVersion",
+            "s3:AbortMultipartUpload",
+            "s3:ListBucket",
+            "s3:GetObject"
+          ],
+          "Effect": "Allow",
+          "Resource": "arn:aws:s3:::BUCKETNAME-HERE"
+        },
+        {
+          "Action": [
+            "s3:PutObject",
+            "s3:PutObjectAcl",
+            "s3:PutObjectAclVersion",
+            "s3:AbortMultipartUpload",
+            "s3:ListBucket",
+            "s3:GetObject"
+          ],
+          "Effect": "Allow",
+          "Resource": "arn:aws:s3:::BUCKETNAME-HERE/*"
+        }
+      ]
+    }
+   ```
 
-## Detection
+4. Update application environment variables
 
-When a PHP application is deployed, we're looking if the `wp-settings.php` file
-is present at the root folder of your app.
+   Then, update your application environment through the Dashboard or with the
+   [Scalingo CLI](http://cli.scalingo.com) `scalingo env-set VARIABLE_NAME=VALUE`:
 
-During the deployment process you'll see the following output, mentioning that
-the framework has correctly been detected.
+   * `DATABASE_URL`: Connection string to the MySQL database - `mysql://localhost:3306/wp-bedrock` - Automatically added with the Scalingo MySQL addon
+   * `WP_ENV`: Set to environment (`development`, `staging`, `production`)
+   * `WP_HOME`: Full URL to WordPress home (https://my-wordpress.osc-fr1.scalingo.io)
+   * `WP_SITEURL`: Full URL to WordPress including subdirectory (https://my-wordpress.osc-fr1.scalingo.io/wp)
+   * `S3_UPLOADS_BUCKET`: Name of the S3 bucket to upload files to
+   * `S3_UPLOADS_KEY`: AWS Access Key ID for S3 authentication
+   * `S3_UPLOADS_SECRET`: AWS Secret Key for S3 authentication
+   * `S3_UPLOADS_REGION`: Region of the S3 bucket
+   * `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT`
 
-```bash
------> Detected WordPress
-...
------> Setting up WordPress
-```
+   You can get some random salts on the [Roots WordPress Salt Generator](https://roots.io/salts.html).
+
+5. Add themes in `web/app/themes` as you would for a normal WordPress site.
+
+6. Add plugins using [Composer](https://getcomposer.org/) and [WordPress Packagist](https://wpackagist.org/search?q=&type=plugin&search=)
+
+   Example for add the `Akismet` plugin:
+
+   ```bash
+   composer require --ignore-platform-reqs wpackagist-plugin/akismet
+   ```
+
+7. Deploy the application on Scalingo
+
+   ```bash
+   # Optionally add theme to your git repository
+   git add web/app/themes
+   git commit -m "Add themes"
+
+   # Then push to Scalingo
+   git push scalingo master
+   ```
+
+8. Access WP admin at `https://my-wordpress.osc-fr1.scalingo.io/wp/wp-admin`
+
+9. Activate the `S3 Uploads` plugin on WP Admin plugins page.
+
+10. That's it.
```

Comment:
```suggestion
```

I don't really like this 10th point as there are already so many points to successfully deploy the WordPress